### ISSUE TITLE
Build Pipelines Failing due to Jest Config Error

### ISSUE THREAD

**Issue by Weltraumschaf** [secureCodeBox/secureCodeBox#1413]
Since this week some hook pipelines are failing consistently with this error:

```
.: 🧪 Starting unit-tests for 'update-field-hook' hook with 'jest@29.1.2'.
● Multiple configurations found:

    * /home/runner/work/secureCodeBox/secureCodeBox/hooks/jest.config.js
    * `jest` key in /home/runner/work/secureCodeBox/secureCodeBox/hooks/package.json

  Implicit config resolution does not allow multiple configuration files.
  Either remove unused config files or select one explicitly with `--config`.

  Configuration Documentation:
  https://jestjs.io/docs/configuration

make[1]: *** [../../test-base.mk:79: unit-test-js] Error 1
make: *** [../../hooks.mk:34: unit-tests] Error 2
Error: Process completed with exit code 2.
```
[Source](https://github.com/secureCodeBox/secureCodeBox/actions/runs/3273084756/jobs/5385009039)

Usually it is the update-field-hook and or cascading-scans-hook. Others are canceled.

First idea was: We use latest version of Jest in our Makefiles, but a pinned version in the package.json. Also we have Jest configs in the config file and in package.json. Maybe new Version dissallow this. We tried to [pin an older version in the Makefiles](https://github.com/secureCodeBox/secureCodeBox/pull/1357/commits/dfcd4a358f658565ed94854eaaba3de9f2634b81), but didn't resolve the issue.

**Comment by RamiSouai** [comment]
[Apparently, multiple config files trigger an error now.](https://github.com/facebook/jest/pull/13356/files/d93a12c47478319a95a203208749b82749fdebf0#r985227596) 

A possible solution would be to merge both config files. Has this been considered?

**Comment by Ilyesbdlala** [comment]
closed by #1415
